crates/muvm-x11bridge/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "muvm-x11bridge"
+version = "0.0.1"
+edition = "2021"
```

```suggestion
name = "muvm-x11bridge"
version = "0.0.1"
authors = [...]
edition = "2021"
rust-version = "1.77.0"
license = "..."
```

Some missing fields that should probably be filled in?

---

I agree. Also, please change the version to `0.1.0` so it's easier to follow SemVer.